ParcelDistributionCenter/ParcelDistributionCenter.Web/Controllers/PackagesController.cs

```diff
@@ -110,6 +147,8 @@ public ActionResult FindPackageByCourierID(string CourierId)
 
             if (model == null)
             {
+                TempData["Message"] = "Package not found.";
```

Comment:
To się już pojawia parę razy. Też będzie to trzeba zwinąć do jakieś metody, albo pola, albo coś. Względem zasady don't repeat yourself

Reply:
Jak znajdę czas to ogarnę to.